icon_validator/rules/plugin_validators/version_bump_validator.py

```diff
@@ -40,31 +41,41 @@ def __init__(self):
         self.local_version = None
         self.remote_version = None
 
+    @staticmethod
+    def is_initial_plugin_version(version_history: List[str]) -> bool:
+        initial_versions = ["0.1.0 - Initial plugin", "1.0.0 - Initial plugin"]
+        if len(version_history) == 1 and any(version in version_history for version in initial_versions):
```

Comment:
for this, would we be better just searching for the term 'Initial plugin' and also `.lower()` it just to catch any potential edge cases

Reply:
Also, it might be worth updating the documentation (or raising a ticket to do it later) for the tooling under the section related to the `create` subcommand to inform the customer they need to include this when making a new plugin

Reply:
I like your idea of using `.lower()` versions to avoid edge cases. Regarding getting rid of the versions, https://docs.rapid7.com/insightconnect/plugin-spec/#version looks like they should be used anyway, what do you think about it? How do you update the docs? What does the process look like?

Reply:
Yeah it seems like we can leave the version in then, that makes sense. Maybe the docs should be updated to say they need to include 'Initial Version' for a new plugin.

To update documentation, the repository for the docs is found here:
https://github.com/rapid7/product-documentation/tree/master/insightconnect/Plugin%20Development